logstash/pipeline/logstash.conf

```diff
@@ -1,7 +1,8 @@
 input {
-	tcp {
-		port => 5000
-	}
+    tcp {
+        port  => 5000
+        codec => json
```

Comment:
Not needed on my use; I believe I'd change this on own fork or so; no reason to make it more complicated in my point of view. 😄